### DISCUSSION TITLE
2023-icpc-seoul-regional/

### DISCUSSION THREAD

**Discussion by giscus** [koderpark/BlogComment#2] [Announcements]
# 2023-icpc-seoul-regional/

안녕하십니까, 제 23회 ICPC 서울리저널 본선에 팀 KookBab 으로 @saywoo, @dk10211과 참여하고 온 코더빡(@koder0205)이라고 합니다.

풀이글 작성하기에는 제 실력이 많이 미숙하다고 판단해 다른 분들이 다뤄주실것이라 생각하고, 그냥 개인적인 경험에 대해서 정리해보고 싶었습니다.

앞서서...

팀 국밥은 구성원 전부가 23학번/04년생 새내기로 구성된 팀으로, 개중에는 사실상 국민대학교 특기자전형의 필수조건인 <국민대학교

https://blog.koder.page/2023-icpc-seoul-regional/

**Comment by usb9245** [comment]
:blobaww:

**Comment by raararaara** [comment]
고생많으셨습니다~!
